source/braille.py

```diff
@@ -1677,11 +1679,13 @@ def previousLine(self, start=False):
 					pass
 				else:
 					dest = dest.obj.makeTextInfo(textInfos.POSITION_LAST)
-					dest.expand(unit)
```

Comment:
Why was this removed?

Reply:
Accidentally. Thanks for pointing it out.